src/modules/crewTypes.ts

```diff
@@ -50,6 +50,7 @@ export interface Vacancy {
 
     activeSince?: Date;
     timeAtWork: number;
+    [key: string]: unknown;
```

Comment:
Why?

Reply:
without it typescript doesn't know that typeof keyof id === 'string'

Reply:
Can be solved with `extends string` and generics